lib/inspec/formatters/base.rb

```diff
@@ -84,6 +85,60 @@ def results
 
     private
 
+    def all_unique_controls
+      unique_controls = Set.new
+      run_data[:profiles].each do |profile|
+        profile[:controls].map { |control| unique_controls.add(control) }
+      end
+
+      unique_controls
+    end
+
+    def statistics
+      failed = 0
+      skipped = 0
+      passed = 0
+      critical = 0
+      major = 0
+      minor = 0
+
+      all_unique_controls.each do |control|
+        next unless control[:results]
+        if control[:results].any? { |r| r[:status] == 'failed' }
+          failed += 1
+          if control[:impact] >= 0.7
```

Comment:
We need to be aware that this introduces a clear definition of what 0.7, 08 etc means. Before we always avoided that and I am not sure if we should rely on that as a core assumptions in inspec.